src/tracks/playabletrack/wavetrack/ui/WaveClipAdjustBorderHandle.cpp

```diff
@@ -437,37 +487,62 @@ UIHandlePtr WaveClipAdjustBorderHandle::HitAnywhere(
         else if (std::abs(px - clipRect.GetRight()) <= BoundaryThreshold * 2)
            leftClip = clip;
     }
-
-    std::unique_ptr<AdjustPolicy> adjustPolicy;
+    
+    std::shared_ptr<WaveClip> adjustedClip;
+    bool adjustLeftBorder {false};
     if (leftClip && rightClip)
     {
-       //between adjacent clips
-       if(ClipParameters::GetClipRect(*leftClip, zoomInfo, rect).GetRight() > px)
-         adjustPolicy = std::make_unique<AdjustClipBorder>(waveTrack, leftClip, false, zoomInfo);//right border
-       else
-         adjustPolicy = std::make_unique<AdjustClipBorder>(waveTrack, rightClip, true, zoomInfo);//left border
+        //between adjacent clips
+        if(ClipParameters::GetClipRect(*leftClip, zoomInfo, rect).GetRight() > px)
```

Comment:
Your new code indents by 4 instead of 3. The way I do it is provisionally copy-paste one `.clang-format` (e.g. that in `lib-stretching-sequence`) in the root directly and format only my changes as I go. I know it's not perfect, but ...